notebooks/simple_xgboost_example.ipynb

```diff
@@ -0,0 +1,752 @@
+    "\n",
+    "For more information on sample configs, please refer this [link](https://github.com/triton-inference-server/server/blob/main/docs/model_configuration.md)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "# Writing config to file\n",
+    "cat > /model_repository/fil/config.pbtxt <<EOL \n",
+    "name: \"fil\"                              # Name of the model directory (fil in our case)\n",
+    "backend: \"fil\"                           # Triton FIL backend for deploying forest models\n",
+    "max_batch_size: 8192\n",
+    "input [\n",
+    " {\n",
+    "    name: \"input__0\"\n",
+    "    data_type: TYPE_FP32\n",
+    "    dims: [ 9 ]                          # Input feature dimensions, in our sample case it's 9\n",
+    "  }\n",
+    "]\n",
+    "output [\n",
+    " {\n",
+    "    name: \"output__0\"\n",
+    "    data_type: TYPE_FP32\n",
+    "    dims: [ 2 ]                          # Output 2 for binary classification model\n",
+    "  }\n",
+    "]\n",
+    "instance_group [{ kind: KIND_GPU }]\n",
+    "parameters [\n",
+    "  {\n",
+    "    key: \"model_type\"\n",
+    "    value: { string_value: \"xgboost\" }\n",
+    "  },\n",
+    "  {\n",
+    "    key: \"predict_proba\"\n",
+    "    value: { string_value: \"false\" }\n",
+    "  },\n",
+    "  {\n",
+    "    key: \"output_class\"\n",
+    "    value: { string_value: \"true\" }\n",
+    "  },\n",
+    "  {\n",
+    "    key: \"threshold\"\n",
+    "    value: { string_value: \"0.5\" }\n",
+    "  },\n",
+    "  {\n",
+    "    key: \"algo\"\n",
+    "    value: { string_value: \"ALGO_AUTO\" }\n",
+    "  },\n",
+    "  {\n",
+    "    key: \"storage_type\"\n",
+    "    value: { string_value: \"AUTO\" }\n",
+    "  },\n",
+    "  {\n",
+    "    key: \"blocks_per_sm\"\n",
+    "    value: { string_value: \"0\" }\n",
+    "  }\n",
+    "]\n",
+    "\n",
+    "EOL"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally, the model repository should look like this:\n",
+    "\n",
+    "```\n",
+    "model_repository/\n",
+    "`-- fil\n",
+    "    |-- 1\n",
+    "    |   `-- xgboost.model\n",
+    "    `-- config.pbtxt\n",
+    "```\n",
+    "\n",
+    "**3. Deploy the model in Triton Inference Server**\n",
+    "\n",
+    "Finally, we can deploy the xgboost model in Triton Inference Server using the following command:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Run the Triton Inference Server in a Subprocess from Jupyter notebook\n",
+    "\n",
+    "# The os.setsid() is passed in the argument preexec_fn so\n",
+    "# it's run after the fork() and before  exec() to run the shell.\n",
+    "triton_process = subprocess.Popen([\"tritonserver --model-repository=/model_repository\"], stdout=subprocess.PIPE, \n",
```

Comment:
```suggestion
    "triton_process = subprocess.Popen([\"tritonserver\", \"--model-repository=/model_repository\"], stdout=subprocess.PIPE, \n",
```

Reply:
Taking care of that should also allow you to eliminate the `shell=True` invocation

Reply:
Ah sorry about this I missed the comma there and it threw an error when I tried to execute it after removing `shell=True` . Updated the command.